racket-mode.el

```diff
@@ -108,6 +108,10 @@ http://www.gnu.org/licenses/ for details.")
      ["Last S-Expression" racket-expand-last-sexp]
      "---"
      ["Again" racket-expand-again])
+    ("Tools"
+     ["Profile" racket-profile]
+     ["Check Syntax" racket-check-syntax-mode]
+     ["Error Trace" racket-errortrace])
```

Comment:
Thanks for doing this. One hitch: The last/third item should be deleted. There's no `racket-errortrace` function.

There is a racket-error-level defcustom variable in racket-custom.el, and the `'high` value corresponds to using errortrace for better stack traces in error messages.

If you're thinking that should be out in a menu, and not buried in customizations? That's a good idea. Unfortunately that would mean writing a few functions, each to set one of the levels, so the menu has something to connect to. And I'm not sure if that's a slippery slope, why not other such settings....